src/components/OptionsList/BaseOptionsList.js

```diff
@@ -125,6 +127,16 @@ function BaseOptionsList({
         flattenedData.current = buildFlatSectionArray();
     });
 
+    useEffect(() => {
+        if (isLoading) {
+            return;
+        }
+
+        // Mark the end of the search page load time. This data is collected only for Search page.
+        Timing.end(CONST.TIMING.OPEN_SEARCH);
+        Performance.markEnd(CONST.TIMING.OPEN_SEARCH);
```

Comment:
If the timer never started given this component is used in other pages too, is that not going to cause any issues?

Reply:
@adhorodyski to confirm, no problem with this right?

Reply:
calling end without a start should be safe, I think